### PR TITLE
Issue #33 add key mapping for file nav

### PR DESCRIPTION
**Pull request by 0sumrich**
# Purpose / Goal
Add key mapping for quick action (basic js) related to files #33

# Type
Please mention the type of PR

[ ] New Feature

First timer here so hope it works as I think it does!

### CODE REVIEW

File: tags/images-slider.tag.html [hunk 1]
```diff
@@ -45,15 +45,38 @@
...
-            })
+                $("#rightpaddle").click(function(e){
+                    tag.slideright(e);
+                })
 
-            $("#rightpaddle").click(function(e){
-                tag.slideright(e);
-            })
+                document.addEventListener('keydown', e => {
+                    console.log(e);
```
Comment (amitguptagwl) [inline tags/images-slider.tag.html]: Please remove console logs

File: tags/images-slider.tag.html [hunk 2]
```diff
@@ -45,15 +45,38 @@
...
+                        tag.slideleft(e);
+                    }
+                    if (e.key == 'ArrowRight'){
+                        tag.slideright(e);
+                    }
+                    if (e.key == 'O' && e.shiftKey){
+                        e.preventDefault();
+                        document.getElementsByClassName('filebutton')[1].click();
+                    }
+                    if (e.key == 'Ø' && e.altKey){
```
Comment (amitguptagwl) [inline tags/images-slider.tag.html]: is it 'O' or zero?

**Comment by 0sumrich** [comment]
Hey no problem, I'll fix that. If you press alt+o it turns it into ø, that
one threw me a bit too.

**Comment by amitguptagwl** [comment]
Oh ok. let me test it.

**Comment by 0sumrich** [comment]
Ok, I have also removed the stray console log for you

**Comment by amitguptagwl** [comment]
There is an issue whether I use o, or Alt + o, or Shift + o all open the dialog to open files. In case of Shift + o, it shows it twice.

**Comment by 0sumrich** [comment]
Thanks Amit. That is strange, I believe the shortcuts should just be doing
whatever the related clicks were/are doing. It might be a mac/windows thing
that is difficult for me to test for - I'm on mac with no windows
environment readily available to test for file browsing issues.

**Comment by amitguptagwl** [comment]
I'm testing on Ubuntu. Can you test it if it is opening the correct window on Mac? Or I can ask someone to test on mac

**Comment by amitguptagwl** [comment]
Never mind, I've fixed the issues. You can check the commit history to know what was missing to do.

**Comment by 0sumrich** [comment]
Cool, thanks, I will do that. Hope I was useful for you

**Comment by amitguptagwl** [comment]
always. I like the people who take initiative, doesn't matter how trivial or difficult it is.

**Comment by amitguptagwl** [comment]
@0sumrich there were some more issues found on this PR. Although I've fixed them, I would appreciate if you go and check the changes. It'll be good for learning.

**Comment by 0sumrich** [comment]
Thanks Amit - had a look and makes sense to me except for one thing - maybe
you can answer? I don't understand the need for 'var tag = this'. If you
use => functions you have access to 'this' anyway. I have quite a lot of
react experience, so riot is new to me, so maybe it's a riot thing. Thanks,
Richard

**Comment by 0sumrich** [comment]
Thanks Amit. I'll keep an eye out if there's anything else I can help out
with. Interesting project. All the best, Richard

**Pull request merged**